pom.xml

```diff
@@ -103,10 +103,9 @@
                 <scope>provided</scope>
             </dependency>
             <dependency>
-                <groupId>org.apache.ant</groupId>
-                <artifactId>ant-compress</artifactId>
-                <version>1.4</version>
-                <type>jar</type>
+                <groupId>net.lingala.zip4j</groupId>
+                <artifactId>zip4j</artifactId>
```

Comment:
Got this from the website 'Zip4j is released under Apache License, Version 2.0. You can get more information on this License here'